lib/rules/no-forbidden-elements.js

```diff
@@ -69,10 +79,26 @@ module.exports = class NoForbiddenElements extends Rule {
 
     throw new Error(errorMessage);
   }
+
+  isInHeadHbsFile(scope, node) {
+    if (scope._filePath === 'app/templates/head.hbs' && node.tag === 'meta') {
+      return true;
+    }
+    return false;
+  }
+
   visitor() {
     return {
-      ElementNode(node) {
+      ElementNode(node, path) {
+        if (hasHeadParent(path)) {
+          return;
+        }
+        if (this.isInHeadHbsFile(this, node)) {
+          return;
+        }
```

Comment:
Hmm. These exceptions only apply for `meta`, not for all forbidden elements. 

Can you add some test cases for:

* other forbidden elements within `<head'>` that still flags an issue?
* other forbidden elements within a filePath for head.hbs

Reply:
If these two conditions stay, they can also be combined.

```suggestion
      ElementNode(node, path) {
        if (hasHeadParent(path) || this.isInHeadHbsFile(this, node)) {
          return;
        }
```